packages/expo-updates/android/build.gradle

```diff
@@ -128,3 +128,22 @@ dependencies {
 repositories {
   mavenCentral()
 }
+
+// [BEGIN] Workaround okhttp/okio compatibility issue
+// Remove when we drop support for SDK 44
+def okhttpPinnedArtifacts = []
+configurations.implementation.getDependencies().removeIf { it ->
+  if (['com.squareup.okhttp3', 'com.squareup.okio'].contains(it.group)) {
+    okhttpPinnedArtifacts.add("${it.group}:${it.name}:${it.version}")
+    return true
+  }
+  return false
+}
+okhttpPinnedArtifacts.each { artifact ->
+  dependencies.add('compileOnly', artifact)
+  dependencies.add('testImplementation', artifact)
```

Comment:
i think we may need `androidTestImplementation` as well since we depend on okhttp in some of the instrumented unit tests in this package

Reply:
the instrumented test executed successfully. i guess it's because okhttp be mocked. however, it's no hurt to add `androidTestImplementation`. i'll add accordingly. thanks @esamelson!